src/WingetCreateCLI/Commands/UpdateCommand.cs

```diff
@@ -1,264 +1,285 @@
+        {
+            Manifests manifests = new Manifests();
+
+            Serialization.DeserializeManifestContents(latestManifestContent, manifests);
+
+            if (manifests.SingletonManifest != null)
+            {
+                manifests = ConvertSingletonToMultifileManifest(manifests.SingletonManifest);
+            }
+
+            VersionManifest versionManifest = manifests.VersionManifest;
+            InstallerManifest installerManifest = manifests.InstallerManifest;
+            DefaultLocaleManifest defaultLocaleManifest = manifests.DefaultLocaleManifest;
+            List<LocaleManifest> localeManifests = manifests.LocaleManifests;
+
+            // Ensure that capitalization matches between folder structure and package ID
+            versionManifest.PackageIdentifier = this.Id;
+            installerManifest.PackageIdentifier = this.Id;
+            defaultLocaleManifest.PackageIdentifier = this.Id;
+            UpdatePropertyForLocaleManifests(nameof(LocaleManifest.PackageIdentifier), this.Id, localeManifests);
+
+            if (!string.IsNullOrEmpty(this.Version))
+            {
+                versionManifest.PackageVersion = this.Version;
+                installerManifest.PackageVersion = this.Version;
+                defaultLocaleManifest.PackageVersion = this.Version;
+                UpdatePropertyForLocaleManifests(nameof(LocaleManifest.PackageVersion), this.Version, localeManifests);
+            }
+
+            if (!this.InstallerUrls.Any())
+            {
+                this.InstallerUrls = installerManifest.Installers.Select(i => i.InstallerUrl).Distinct().ToArray();
+            }
+
+            // We only support updates with same number of installer URLs
+            if (this.InstallerUrls.Distinct().Count() != installerManifest.Installers.Select(i => i.InstallerUrl).Distinct().Count())
+            {
+                Logger.ErrorLocalized(nameof(Resources.MultipleInstallerUpdateDiscrepancy_Error));
+                return null;
+            }
+
+            foreach (var url in this.InstallerUrls)
+            {
+                string packageFile = await DownloadPackageFile(url);
+                if (string.IsNullOrEmpty(packageFile))
+                {
+                    return null;
+                }
+
+                this.packageFiles.Add(packageFile);
+            }
+
+            if (!PackageParser.UpdateInstallerNodes(installerManifest, this.InstallerUrls, this.packageFiles, out List<WingetCreateCore.Models.Installer.Installer> installersMissingMatches))
+            {
+                if (installersMissingMatches.Any())
+                {
+                    Logger.ErrorLocalized(nameof(Resources.MultipleInstallerUpdateDiscrepancy_Error));
+                    foreach (var installer in installersMissingMatches)
+                    {
+                        Logger.ErrorLocalized(nameof(Resources.MissingPackageError_Message), installer.InstallerType, installer.Architecture);
```

Comment:
>Resources.MissingPackageError_Message [](http://example.com/codeflow?start=53&length=37)

Make sure to add a comment in the resources file explaining what {0} and {1} will be replaced with.